data/json/professions.json

```diff
@@ -3417,5 +3417,368 @@
+            "points": 3,
+            "CBMs":[
+                "bio_batteries",
+                "bio_power_storage",
+                "bio_int_enhancer",
+                "bio_memory"
+            ],
+            "items": {
+                "both": [
+                    "dress_shirt",
+					"jacket_light",
+					"pants",
+					"socks",
+					"dress_shoes",
+					"skinny_tie",
+					"tieclip",
+					"fancy_sunglasses",
+					"knit_scarf",
+					"wristwatch",
+					"mbag",
+					"water_mineral",
+					"pda",
+					"cell_phone",
+					"money_bundle"
+                ],
+                "male": [
+                    "briefs"
+                ],
+                "female": [
+                    "panties"
+                ]
+            }
+    },
+	{
+        "type": "profession",
+        "ident": "dodgeball_player",
+        "name": "Dodgeball Player",
+        "description": "You liked to play dodgeball, where failing to dodge the ball meant you were out. Now failing to dodge threatens your life. Don't slip up.",
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "dodge"
+            },
+            {
+                "level": 1,
+                "name": "throw"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+				"jacket_light",
+				"jeans",
+				"socks",
+				"lowtops",
+				"runner_bag",
+				"juice"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "boy_shorts"
+            ]
+        }
+    },
+	{
+        "type": "profession",
+        "ident": "science_club_mem",
+        "name": "Science Club Member",
+        "description": "You were a member of the school science club, and right now you're as upset as you've ever been that the school wouldn't let you play with the really fun chemicals that make things go boom. At least now no one's around to tell you that you can't.",
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "cooking"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+				"jacket_light",
+				"jeans",
+				"socks",
+				"sneakers",
+				"knit_scarf",
+				"glasses_safety",
+				"chemistry_set",
+				"textbook_chemistry"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "panties"
+            ]
+        }
+    },
+	{
+        "type": "profession",
+        "ident": "av_club_mem",
+        "name": "A/V Club Member",
+        "description": "You were a member of the school A/V club. Are those even a thing in 2042? You're sure there's some way you can use your technical skills to help stay alive. You just haven't figured out how to make an awesome death ray yet.",
```

Comment:
Maybe omit the rhetorical question ('Are those even a thing in 2042?'). Clearly, they are, else this profession wouldn't exist.